src/main/java/org/embulk/output/sf_bulk_api/UserPasswordConnectorConfigCreater.java

```diff
@@ -0,0 +1,25 @@
+package org.embulk.output.sf_bulk_api;
+
+import com.sforce.soap.partner.PartnerConnection;
+import com.sforce.ws.ConnectionException;
+import com.sforce.ws.ConnectorConfig;
+
+public class UserPasswordConnectorConfigCreater implements ConnectorConfigCreater
+{
+    final PluginTask pluginTask;
+
+    UserPasswordConnectorConfigCreater(final PluginTask pluginTask)
+    {
+        this.pluginTask = pluginTask;
+    }
+
+    public ConnectorConfig createConnectorConfig() throws ConnectionException
+    {
+        ConnectorConfig config = new ConnectorConfig();
+        config.setUsername(pluginTask.getUsername().get());
+        config.setPassword(pluginTask.getPassword().get() + pluginTask.getSecurityToken().get());
+        config.setAuthEndpoint(pluginTask.getAuthEndPoint().get() + pluginTask.getApiVersion());
```

Comment:
In original code, auth endpoint has suffix `"/"`.
```
config.setAuthEndpoint(pluginTask.getAuthEndPoint() + pluginTask.getApiVersion() + "/");

```
https://github.com/trocco-io/embulk-output-sf_bulk_api/pull/7/files#diff-69a8326a0108a38b658e5dd88c52994daca3cf96f17893353797c427a4bfdce2L63

No need to have the suffix `"/"`?